tests/ClientTest.php

```diff
@@ -1673,6 +1673,26 @@ public function testSetAllObjectSerialize()
         $this->assertFalse($client->getSerializer()->getAllObjectSerialize());
         $this->assertFalse($client->getReprSerializer()->getAllObjectSerialize());
     }
+
+    public function testClearBreadcrumb()
```

Comment:
This is not needed as the `Recorder` class is already unit tested, so this is useless. Maybe you should test that the `leaveBreadcrumb` method calls the recorder, but I don't know if it's worth it or was already done somewhere else

Reply:
I don't test `Recorder::clear`, I'm testing `Client::clearBreadcrumbs`

Reply:
The `Client::clearBreadcrumbs` method just calls the `Recorder::clear` method and this one is already tested, so what's the advantage of adding this test?

Reply:
Erm... To test that `Client::clearBreadcrumbs` really calls `Recorder::clear`?

Reply:
Then there is no need imho to duplicate the whole block of code that tests that breadcrumbs are removed, you could just check that the `Recorder::clear` method is called and let the unit tests of that class catch any error if something doesn't work inside it 😃 The point is that imho this whole block of code is just a duplication of https://github.com/getsentry/sentry-php/blob/dcc287f6579b765c7935e0e60fa642ae029b834f/tests/Breadcrumbs/RecorderTest.php#L73-L90

Reply:
> you could just check that the `Recorder::clear` method is called

How? With DI? Of course, but I though it is easy to check empty/not empty instead of injecting an object.